integration/golden/curl-2lo.golden

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what is purpose of this file?

Reply:
This is the expected output of a successful curl 2LO test case.